design/MetricProfile.md

```diff
@@ -0,0 +1,94 @@
+# Create Metric Profile
+
+Client will invoke the Kruize createMetricProfile API for each deployment. Documentation still in progress stay
```

Comment:
Not for each deployment can you please refer this link and rephrase above accordingly
https://docs.google.com/document/d/12CHxowvyyOTUeBXLg2K4NeE7Q4PFuVO-pjHr4cgXWcs/edit

Reply:
Combine MetricProfile and MetricProfileAPI into a single file and include a reference in MonitotingModeAPI.md.

Reply:
As discussed included MetricProfile reference in `KruizeLocalAPI.md` specific to local monitoring